src/index.tsx

```diff
@@ -41,6 +41,11 @@ export namespace CommandIDs {
 export const NotebookJobsPanelId = 'notebook-jobs-panel';
 export { Scheduler } from './tokens';
 
+type EventLog = {
+  body: { name: string; error?: { message?: string; httpStatusCode?: number } };
```

Comment:
Capturing the error detail/string should be sufficient to capture the error.
```suggestion
  body: { name: string; detail?: string } };
```